metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -293,6 +293,7 @@ func NewContainerMetadataEnricher(
 			podStore, _ := nodeStore.AddPodStore(podId)
 
 			for _, container := range append(pod.Spec.Containers, pod.Spec.InitContainers...) {
```

Comment:
I know we have a for here, but just doublehcecking if we can make a test wit 3 containers. In case we miss sth

Reply:
I can reassure you it works because I tested it!